src/frontend/dash/circlePacker.ts

```diff
@@ -21,6 +28,7 @@ export class MemberCircle {
     x: number;
     y: number;
     r: number;
+    // touching: MemberCircle[] = [];
```

Comment:
There's a lot of commented out code like this. If it's no longer used, it should probably be gotten rid of to avoid confusion.